### PR TITLE
Data migration

### PR DESCRIPTION
**Pull request by dani-efr**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,5 @@ studies/data/*.xlsx
 uncontrast_studies/data/*.csv
 uncontrast_studies/data/*.xls
 uncontrast_studies/data/*.xlsx
+/uncontrast_studies/data/logs
+/uncontrast_studies/data/old datasets
```
Comment (alonisser) [inline .gitignore:27]: this is the folder? why with a space.? generally a bad practice to use folder with spaces

File: configuration/uncontrast_initial_data_types.py [hunk 2]
```diff
@@ -36,22 +36,28 @@
     "Discrimination",
     "Detection",
     "Task is irrelevant to the effect",
```
Comment (alonisser) [inline configuration/uncontrast_initial_data_types.py:38]: do changes here mean we need to remigrate?

File: uncontrast_studies/management/commands/load_uncon_data.py [hunk 3]
```diff
@@ -27,15 +28,26 @@
...
+current_date = date.today()
+ERROR_LOG_PATH = f"uncontrast_studies/data/logs/uncontrast_errors_log_{current_date}.xlsx"
 
 
 class Command(BaseCommand):
     help = "Load uncon existing data"
 
     def handle(self, *args, **options):
         # Read .xlsx file and convert to dict
+        # maor_experiments_data = get_list_from_excel(MAOR_FILE_PATH, sheet_name="experiments")
```
Comment (alonisser) [inline uncontrast_studies/management/commands/load_uncon_data.py:44]: why is this comment here? for later?

**Review by alonisser — APPROVED** [review]

File: configuration/uncontrast_initial_data_types.py (hunk 2 above)
Comment (dani-efr) [reply configuration/uncontrast_initial_data_types.py:38]: yes, as this is still a process of finding out what's missing here. Every time we load and reload the data we need to run existing migrations anyway. But we don't have to **make** the migrations. When I finish with the data corrections there will be a final version with no changes.

File: uncontrast_studies/management/commands/load_uncon_data.py (hunk 3 above)
Comment (dani-efr) [reply uncontrast_studies/management/commands/load_uncon_data.py:44]: Francois wanted to work on a separate spreadsheet for the corrections process so I keep that option here as a comment until I get his files. it's temporary

**Pull request merged**